packages/@core.keystone/test.utils.js

```diff
@@ -410,6 +414,30 @@ const getSchemaObject = async (schemaList, fields, where) => {
     return result.data.obj
 }
 
+async function waitFor (callback, options = { timeout: 2000, interval: 50 }) {
+    const timeout = options.timeout || 5000
+    const interval = options.interval || 50
+    let savedError = null
+
+    return new Promise((res, rej) => {
```

Comment:
`res` is confusing with result. please user `resolve` & `reject`

Reply:
it's a common style for promises